### ISSUE TITLE
training loss curve

### ISSUE THREAD

**Issue by Kirayue** [zisianw/FaceBoxes.PyTorch#41]
I tried to implement faceboxes from scratch, I took this repo as a reference. And I wondered what's the training loss curve looks like. My training curve has many impulses after some epochs and decreases to 5 or 4 again. Could you give me some suggestions?

![image](https://user-images.githubusercontent.com/13309212/56117636-1184db80-5f9b-11e9-82ff-e825e3a04dbc.png)

**Comment by xsacha** [comment]
Training loss of this repo starts at 10, moves to about 3 after 30 epochs and just stays at 3.
